bloom-housing/bloom
# feat: lottery public list view

This PR addresses #4053 


## Description

This PR adds the final touches to the applications lottery view of the public side. This mainly includes custom tags and dates. It also includes some string twekas (@eajensenwa approved the "View application" vs "View your application" tweak)

## How Can This Be Tested/Reviewed?

This can be tested by signing into the public site and applying to a few different listings. Then, update the listings to show as open without a due date, open with a due date, closed without a lottery date, closed with a lottery date, and lottery ran.

Additionally ensure all tests have passed below.
## Author Checklist:


## Review Process:

- Read and understand the issue
- Ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Either (1) explicitly ask a clarifying question, (2) request changes, or (3) approve the PR, even if there are very small remaining changes, if you don't need to re-review after the updates
